cmake/std/atdm/sems-rhel6/environment.sh

```diff
@@ -44,9 +98,8 @@ module purge
 module load sems-env
 module load sems-git/2.10.1
 
-module load atdm-env
```

Comment:
did you intend to remove this completely?

Reply:
@rppawlo, thanks for the review and approval!

Yes we meant to remove `atdm-env` because now we are using CMake and Ninja from the SEMS env as shown at:

* https://github.com/bartlettroscoe/Trilinos/blob/6fa6ae321f89129fe3586b4310f0fb7c9239e8ff/cmake/std/atdm/sems-rhel6/environment.sh#L101
* https://github.com/bartlettroscoe/Trilinos/blob/6fa6ae321f89129fe3586b4310f0fb7c9239e8ff/cmake/std/atdm/sems-rhel6/environment.sh#L102

Before, we were using these from the ATDM env because SEMS did not have updated CMake or Fortran-capable ninja.

Thanks again!